packages/web-app-files/src/components/SideBar/Shares/FileShares.vue

```diff
@@ -198,17 +198,20 @@ export default defineComponent({
     },
 
     collaborators() {
-      return [...this.outgoingCollaborators].sort(this.collaboratorsComparator).map((c) => {
-        const collaborator: typeof c & { key?: string; resharers?: User[] } = { ...c }
-        collaborator.key = 'collaborator-' + collaborator.id
-        if (
-          collaborator.owner.name !== collaborator.fileOwner.name &&
-          collaborator.owner.name !== this.user.id
-        ) {
-          collaborator.resharers = [collaborator.owner]
-        }
-        return collaborator
-      })
+      return [...this.outgoingCollaborators]
+        .filter((c) => c.collaborator.displayName)
+        .sort(this.collaboratorsComparator)
+        .map((c) => {
+          const collaborator: typeof c & { key?: string; resharers?: User[] } = { ...c }
+          collaborator.key = 'collaborator-' + collaborator.id
+          if (
+            collaborator.owner.name !== collaborator.fileOwner.name &&
+            collaborator.owner.name !== this.user.id
+          ) {
+            collaborator.resharers = [collaborator.owner]
+          }
+          return collaborator
+        })
```

Comment:
I'd prefer adding the filter to `outgoingCollaborators` in our `useShare` composable.

I guess this is CERN specific? We can also think about overwriting the composable via cern config [like we did here](https://github.com/owncloud/web/pull/9438/commits/4bd2760355a0fa6e9b055a21518b45ef68376e4f). Although it's such a small change, it's probably not worth the effort 🤔

Reply:
I'm open to doing that, the benefit of doing it here is that one could also add some sort of hint to the UI about further shares existing but the webUI not being able to correctly render these (whcih technically could also be done inside the composable and then exported, ofc). Let me discuss this with @tbsbdr